metapool/rescale_counts.py

```diff
@@ -0,0 +1,67 @@
+import pandas as pd
+import numpy as np
+
+
+def _to_column_percentage(df):
+    return df / df.sum() * 100
+
+
+def _to_row_percentage(df):
+    return df.div(df.sum(axis=1), axis=0) * 100
+
+
+def _to_log10_cpm(df):
+    aligned_reads = df.sum()
+    df_log10_cpm = np.log10(df / aligned_reads * 1000000)
+    return df_log10_cpm
+
+
+def _apply_model(df_log10_cpm, linear_models):
+    cols = []
+    failed_colnames = []
+    for c in df_log10_cpm:
+        r = linear_models[linear_models["sample_name"] == c]
+        if r.empty:
+            # TODO FIXME HACK:  What do we do if this sample can't be rescaled
+            #  because no linear model was created?
+            failed_colnames.append(c)
+            continue
+        converted = r["a_intercept"].iloc[0] + df_log10_cpm[c] * \
+            r["b_slope"].iloc[0]
+        converted = np.power(10, -converted)
+        cols.append(converted)
+
+    converted_reads = pd.DataFrame(data=cols).T
+    return converted_reads, failed_colnames
+
+
+def convert_read_count_to_cell_count_per_g_input(df,
+                                                 metadata_features,
+                                                 prep_info_samples):
+    lengths = metadata_features.loc[df.index, "total_length"]
+    sample_weights = prep_info_samples.loc[df.index,
```

Comment:
As far as I can tell, this is the problematic line (why the tests are failing), and looking at the code I think what's happening is that the values in `df` are transversed: basically, it's looking by features but it should be looking by sample. @charles-cowart, can you take a look?